traffic/data/adsb/decode.py

```diff
@@ -1626,7 +1629,7 @@ def aircraft(self) -> list[Dict[str, Any]]:
             (
                 dict(
                     icao24=key,
-                    callsign=ac.callsign,
+                    callsign=ac.callsign or key,
```

Comment:
Tu confirmes que ça veut dire `ac.callsign if ac.callsign is not None else key` ?

Reply:
Je confirme :)